File: iniciante/1072/1072.js
// 1072 - Intervalo 2 - beecrowd

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const n = parseInt(lines[0]);
let inn = 0;
let out = 0;

for (let i = 0; i < n; i++) {
	let x = parseInt(lines[i + 1]);
	if (x >= 10 && x <= 20){ inn++; }
	else { out++; }
}

console.log(`${inn} in`);
console.log(`${out} out`);

File: iniciante/1072/1072.c
// 1072 - Intervalo 2 - beecrowd

#include <stdio.h>

int main(void)
{
	int N; scanf("%d", &N);
	
	int x;
	int contIn = 0, contOut = 0;


	for (int i = 0; i < N; i++)
	{
		scanf("%d", &x);
		if (x >= 10 & x <= 20) contIn++;
		else contOut++;
	}

	printf("%d in\n", contIn);
	printf("%d out\n", contOut);

	return 0;
}
